x/axelarnet/keeper/msg_server.go

```diff
@@ -159,12 +159,19 @@ func (s msgServer) ConfirmDeposit(c context.Context, req *types.ConfirmDepositRe
 		return nil, err
 	}
 
-	s.Logger(ctx).Info(fmt.Sprintf("deposit confirmed to %s with transfer ID %d", req.DepositAddress.String(), transferID))
+	recipient, ok := s.nexus.GetRecipient(ctx, depositAddr)
+	if !ok {
+		return nil, fmt.Errorf("no recipient linked to deposit address %s", req.DepositAddress.String())
+	}
+
+	s.Logger(ctx).Info(fmt.Sprintf("deposit confirmed for %s on chain %s to recipient %s on chain %s for asset %s with transfer ID %d", req.DepositAddress.String(), exported.Axelarnet.Name, recipient.Address, recipient.Chain.Name, amount.String(), transferID))
```

Comment:
use facets instead

Reply:
I can add facets for these also, but I think we should switch from log message content to facets everywhere in the same release. Otherwise, the debugging experience will be painful.

Reply:
why would it be painful?

Reply:
Because currently for debugging core logs, I search the deposit address directly rather than with a facet to get the relevant logs. Our mods are also used to debugging in a similar manner.

So, I think we should add facets everywhere before removing the info from the log content for consistency. And do some education around debugging once that occurs, especially since people not familiar with core already find debugging difficult.